packages/contentful/migrations/calendars/20230328165340-create-calendars-content-model.js

```diff
@@ -0,0 +1,150 @@
+module.exports.description = 'Create calendars content model';
+
+module.exports.up = (migration) => {
+  const calendars = migration
+    .createContentType('calendars')
+    .name('Calendars')
+    .description('')
+    .displayField('name');
+  calendars
+    .createField('name')
+    .name('Name')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  calendars
+    .createField('googleCalendarId')
+    .name('Google Calendar ID')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        unique: true,
+      },
+      {
+        size: {
+          min: 8,
+        },
+      },
+      {
+        regexp: {
+          pattern:
+            '^[a-zA-Z0-9.!#$%&’*+\\\\/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:.[a-zA-Z0-9-]+)*$',
+          flags: null,
+        },
+      },
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  calendars
+    .createField('color')
+    .name('Color')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([
+      {
+        in: [
+          '#B1365F',
+          '#5C1158',
+          '#711616',
+          '#691426',
+          '#BE6D00',
+          '#B1440E',
+          '#853104',
+          '#8C500B',
+          '#754916',
+          '#88880E',
+          '#AB8B00',
+          '#856508',
+          '#28754E',
+          '#1B887A',
+          '#28754E',
+          '#0D7813',
+          '#528800',
+          '#125A12',
+          '#2F6309',
+          '#2F6213',
+          '#0F4B38',
+          '#5F6B02',
+          '#4A716C',
+          '#6E6E41',
+          '#29527A',
+          '#2952A3',
+          '#4E5D6C',
+          '#5A6986',
+          '#182C57',
+          '#060D5E',
+          '#113F47',
+          '#7A367A',
+          '#5229A3',
+          '#865A5A',
+          '#705770',
+          '#23164E',
+          '#5B123B',
+          '#42104A',
+          '#875509',
+          '#8D6F47',
+          '#6B3304',
+          '#333333',
+        ],
+      },
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  calendars
+    .createField('syncToken')
+    .name('Google Last Sync Token')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(true)
```

Comment:
When disable is true, the field does not appear on Contentful

![calendar-disabled-field](https://user-images.githubusercontent.com/16595804/228581296-590a41aa-9f86-4154-9177-81b9d964250b.png)

But they still can come in the response
![calendar-graphql](https://user-images.githubusercontent.com/16595804/228581421-3633f302-e2be-407b-97bc-c0913bcc026d.png)

In this case, I've left the field enabled, added a value to the entry and then disabled the field. That's why it has a value.

Reply:
isnt that what we want? that the field is not visible in the UI but still returned in the API? I mean it's quite handy to have these field in squidex for debugging but we can always just use the API for that. I don't know, what do you think?

Reply:
oh sorry, you're just saying that you did in fact make it disabled but just explaining how you got the value into it yeah? sure that makes sense, it would only ever be updated via the API so this works for me

Reply:
yeah, I was just explaining why the value does not appear in the ui but appears in the response :)